### PR TITLE
Disables Pour it Again issue #624

### LINKED ISSUE #624: Using Pour it Again on Removed Command
Using ```Pour it Again``` button on command that has been removed or had name changed, the page navigates to an unpopulated command view page, and does indicate to user the command no longer exists.

### PR DESCRIPTION
**Pull request by west270**
Disables `Pour it Again` button if system or command is not found.

Fixes #624

### CODE REVIEW

File: src/ui/src/js/controllers/request_view.js [hunk 1]
```diff
@@ -13,6 +13,7 @@ requestViewController.$inject = [
   'RequestService',
   'SystemService',
   'EventService',
+  'NamespaceService',
```
Comment (TheBurchLog) [inline src/ui/src/js/controllers/request_view.js:16]: Unused import `NamespaceService`

File: src/ui/src/js/controllers/request_view.js [hunk 2]
```diff
@@ -38,7 +39,8 @@ export default function requestViewController(
     localStorageService,
     RequestService,
     SystemService,
-    EventService) {
+    EventService,
+    NamespaceService) {
```
Comment (TheBurchLog) [inline src/ui/src/js/controllers/request_view.js:43]: Unused import `NamespaceService`

**Review by TheBurchLog — COMMENTED** [review]
Looks good, minus the few changes suggested. Once you get those fixed go ahead and merge this in.

**Review by TheBurchLog — APPROVED** [review]

File: src/ui/src/js/controllers/request_view.js [hunk 3]
```diff
@@ -169,7 +171,26 @@ export default function requestViewController(
   $scope.successCallback = function(request) {
     $scope.request = request;
     $scope.filename = $scope.request.id;
-
+    let namespace = $scope.request.namespace || $scope.config.gardenName
```
Comment (hazmat345) [inline src/ui/src/js/controllers/request_view.js:174]: ;

File: src/ui/src/js/controllers/request_view.js [hunk 4]
```diff
@@ -169,7 +171,26 @@ export default function requestViewController(
...
+    let request_system = SystemService.findSystem(namespace, $scope.request.system,
+                         $scope.request.system_version);
+    if (request_system != undefined) {
+        let commands = request_system.commands;
+        for (let i = 0; i < commands.length; i++) {
+            if (commands[i].name == request.command){
+                $scope.disabledPourItAgain = false;
+                $scope.msgPourItAgain = null
```
Comment (hazmat345) [inline src/ui/src/js/controllers/request_view.js:182]: ;

File: src/ui/src/js/controllers/request_view.js [hunk 5]
```diff
@@ -169,7 +171,26 @@ export default function requestViewController(
...
+                break;
+            }
+            else {
+                $scope.disabledPourItAgain = true;
+                $scope.msgPourItAgain = 'Unable to find command'
```
Comment (hazmat345) [inline src/ui/src/js/controllers/request_view.js:187]: ;

File: src/ui/src/js/controllers/request_view.js [hunk 6]
```diff
@@ -169,7 +171,26 @@ export default function requestViewController(
...
+            }
+        }
+    } else {
+        $scope.disabledPourItAgain = true
+        $scope.msgPourItAgain = 'Unable to find system'
```
Comment (hazmat345) [inline src/ui/src/js/controllers/request_view.js:192]: ;

**Review by hazmat345 — CHANGES_REQUESTED** [review]
Some missing semicolons. Other than that looks good and works for me.

**Review by hazmat345 — APPROVED** [review]
LGTM :+1:

**Pull request merged**